src/main/java/org/example/servletsHomework/servlet/MyContextListener.java

```diff
@@ -0,0 +1,56 @@
+package org.example.servletsHomework.servlet;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.servlet.*;
+import jakarta.servlet.http.HttpFilter;
+import jakarta.servlet.http.HttpServlet;
+import org.example.servletsHomework.dao.TextDao;
+import org.example.servletsHomework.dao.HashmapTextDao;
+import org.example.servletsHomework.filters.TextFilterIfUserExist;
+import org.example.servletsHomework.model.IdGenerator;
+import org.example.servletsHomework.model.RealIdGenerator;
+import org.example.servletsHomework.service.TextService;
+import org.example.servletsHomework.storage.TokensAndUserStorage;
+
+
+import java.util.EnumSet;
+
+
+public class MyContextListener implements ServletContextListener {
+
+
+    @Override
+    public void contextInitialized(ServletContextEvent sce) {
+
+        ServletContext servletContext = sce.getServletContext();
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        IdGenerator idGenerator = new RealIdGenerator();
+        TextDao textDao = new HashmapTextDao();
+        TextService textService = new TextService(textDao);
+        TokensAndUserStorage tokensAndUserStorage = new TokensAndUserStorage();
+
+
+        HttpFilter filterIfUserExist = new TextFilterIfUserExist(tokensAndUserStorage);
+        HttpFilter filterIfTokenValid = new TextFilterIfUserExist(tokensAndUserStorage);
+
+        HttpServlet registerServlet = new RegisterServlet(tokensAndUserStorage);
+        HttpServlet loginServlet = new LoginServlet(tokensAndUserStorage);
+        HttpServlet textServlet = new TextsServlet(objectMapper, idGenerator, textService, tokensAndUserStorage);
+
+        String url = "/texts/*";
+
+        FilterRegistration.Dynamic textFilterUserExist = servletContext.addFilter("textFilterUserExist", filterIfUserExist);
+        textFilterUserExist.addMappingForUrlPatterns(EnumSet.of(DispatcherType.REQUEST), true, url);
```

Comment:
Just imagine that you had task to provide authorization/authentification for your application. I'm as another developer need to introduce new endpoints : "/articles/".  Found that I no longer need get token to achieve "/articles/". This is decently security bug and testers returned this task to me. How can you make life of other developer easier ? How logic regarding tokens, registrations will be different in case someone will create another controller ?